.github/workflows/publish-npm.yml

```diff
@@ -26,6 +26,10 @@ on:
         required: true
         type: string
         default: itwinui-css
+      tag:
+        required: true
```

Comment:
If this was `required: false`, then I think you wouldn't need to pass the `tag` in release-please.yml because it already has a default value.

But since this is already tested working, I'm fine with it.